.github/workflows/release-container.yaml

```diff
@@ -0,0 +1,66 @@
+name: Release Container Image
+
+on:
+  push:
+    tags:
+      - 'wallet-v[0-9]+.[0-9]+.[0-9]+' # support v1.2.3
+      - 'wallet-v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+' #supports release-candidate versions v1.2.3-rc.4
+
+jobs:
+
+  define-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.output-version.outputs.version }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set env
+        id: output-version
+        run: echo "version=${GITHUB_REF#refs/*/v}" >> $GITHUB_OUTPUT
+
+  publish-server-container:
+    name: Publish Wallet Server container
+    runs-on: ubuntu-20.04
+    needs:
+      - define-version
+    env:
+      project: src/ProjectOrigin.Wallet.Server
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: ./src
+          file: ./src/ProjectOrigin.Wallet.Server/Dockerfile
+          push: true
+          tags: ghcr.io/project-origin/wallet-server:${{ needs.define-version.outputs.version }}
+
+  create-release:
+    name: Create GitHub Release
+    runs-on: ubuntu-latest
+    needs:
+      - publish-client-nuget
```

Comment:
There is no `publish-client-nuget` job, so I do not think that `create-release` will be able to run

Reply:
has been fixed